### PR TITLE
Use querystring prop in ListingBody

### PR DESCRIPTION
**Pull request by giuliaghisini**
New blocks schema adds **querystring** prop to listing block data, but it was not considered in listing body. 
Added compatibility for new block schema and  backwards compatibility

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: src/components/manage/Blocks/Listing/ListingData.jsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ const ListingData = (props) => {
         onChangeBlock(block, {
           ...data,
           [id]: value,
-          // For backwards compat, we keep the content of the querystring widget spreaded
-          // but we also keep the data in the place it belongs, for future deprecation
-          ...(id === 'querystring' && { ...value }),
+          // // For backwards compat, we keep the content of the querystring widget spreaded
+          // // but we also keep the data in the place it belongs, for future deprecation
+          // ...(id === 'querystring' && { ...value }),
```
Comment (sneridagh) [inline src/components/manage/Blocks/Listing/ListingData.jsx:22]: If we deprecate this, let's remove it completely.

**Review by sneridagh — APPROVED** [review]
LGTM, let's move the migration step before we ship this.

**Comment by sneridagh** [comment]
@cekk could you please take a look? 

https://github.com/kitconcept/kitconcept.volto/blob/71e71f9f741deee7b483d2611728efb2a426a22e/src/kitconcept/volto/upgrades.py#L6-L64

**Comment by cekk** [comment]
> @cekk could you please take a look?
> 
> https://github.com/kitconcept/kitconcept.volto/blob/71e71f9f741deee7b483d2611728efb2a426a22e/src/kitconcept/volto/upgrades.py#L6-L64

The code looks good. I've also tried it locally on a real db and the blocks seems migrated correctly. I'm waiting final review from a real Volto 13 environment from @giuliaghisini

**Comment by giuliaghisini** [comment]
> > @cekk could you please take a look?
> > https://github.com/kitconcept/kitconcept.volto/blob/71e71f9f741deee7b483d2611728efb2a426a22e/src/kitconcept/volto/upgrades.py#L6-L64
> 
> The code looks good. I've also tried it locally on a real db and the blocks seems migrated correctly. I'm waiting final review from a real Volto 13 environment from @giuliaghisini

It works well! now we are ready

**Pull request merged**